public/js/hipsterList.js

```diff
@@ -0,0 +1,114 @@
+
+function HipsterList() {
+//menu items
+ this.shopInfo =
+      {
+        "shopName": "The Coffee Connection",
+        "address": "123 Lakeside Way",
+        "phone": "16503600708",
+        "prices": 
+          {
+            "Cafe Latte": 4.75,
+            "Flat White": 4.75,
+            "Cappucino": 3.85,
+            "Single Espresso": 2.05,
+            "Double Espresso": 3.75,
+            "Americano": 3.75,
+            "Cortado": 4.55,
+            "Tea": 3.65,
+            "Choc Mudcake": 6.40,
+            "Choc Mousse": 8.20,
+            "Affogato": 14.80,
+            "Tiramisu": 11.40,
+            "Blueberry Muffin": 4.05,
+            "Chocolate Chip Muffin": 4.05,
+            "Muffin Of The Day": 4.55
+          }
+      };
+    this.list = [];
+    this.price = [];
+    this.total = [];
+    this.quantity = [];
+    this.payment = [];
+
+}
+
+//functions for list
```

Comment:
you don't need comments like this, you're coding in a standard way so people would expect the functions to be here, only comment surprising behaviour